bin/test

```diff
@@ -110,16 +121,16 @@ function getDownloadUrl(clb) {
     }
 
     getTag(function (tag) {
-        return clb(['https://vscode-update.azurewebsites.net', tag, downloadPlatform, 'stable'].join('/'));
+        return clb(['https://vscode-update.azurewebsites.net', tag, downloadPlatform, (isInsiders ? 'insider' : 'stable')].join('/'));
```

Comment:
If you aren't a fan for the ? operator I can move it into a method and re-use that below as well.